admin-dev/themes/default/js/calendar.js

```diff
@@ -307,9 +307,12 @@ function setPreviousYear() {
   $('#date-end-compare').val(endDate.format($('#date-start').data('date-format')));
 }
 
+var datepickerStart;
+var datepickerEnd;
```

Comment:
```suggestion
let datepickerStart;
let datepickerEnd;
```

When these vars are out of the scope, do it works with let? I think so